app/src/main/java/com/account/service/AccountRoleValidator.java

```diff
@@ -0,0 +1,40 @@
+package com.account.service;
+
+import java.util.List;
+import java.util.UUID;
+
+import org.springframework.stereotype.Service;
+
+import com.account.exception.BadRequestException;
+import com.account.exception.NotFoundException;
+import com.account.model.AccountRoleRequestModel;
+import com.account.persistence.entity.AccountRole;
+import com.account.persistence.repository.AccountRoleRepository;
+
+import lombok.RequiredArgsConstructor;
+
+@Service
+@RequiredArgsConstructor
+public class AccountRoleValidator {
+
+    private final AccountRoleRepository accountRoleRepository;
+
+    public UUID validateAccountRoleRequestData(UUID accountID, AccountRoleRequestModel accountRoleRequestModel) {
```

Comment:
here, you are performing validation not, looking for some id. Remember, Single Responsibility principal 
Conceptually, this validator is not needed